e2e/ansible/playbooks/hyperconverged/test-k8s-memleak/memleak.yml

```diff
@@ -115,12 +115,12 @@
          delegate_to: "{{groups['kubernetes-kubeminions'].0}}"
 
        - name: Place data using dd
-         shell: timeout 200s dd if=/dev/urandom of={{mount_point}}/f1 bs=512k count=10000 &
+         shell: "timeout 200s dd if=/dev/urandom of={{mount_point}}/f1 bs={{block_size}} count={{count_num}} oflag=dsync &"
          become: true
          delegate_to: "{{groups['kubernetes-kubeminions'].0}}"
 
        - name: Place data using dd
-         shell: timeout 200s dd if=/dev/urandom of={{mount_point}}/f2 bs=1m count=10000 &
+         shell: "timeout 200s dd if=/dev/urandom of={{mount_point}}/f2 bs={{block_size1}} count={{count_num}} oflag=dsync &"
          become: true
          delegate_to: "{{groups['kubernetes-kubeminions'].0}}"
 
```

Comment:
Both the dd commands can be invoked from within the same shell task using the ansible shell module's ("|") construct. This will avoid multiple SSH/python file transfers during the playbook run.

Reply:
@ksatchit Implemented review comments. Can you please review?